docs/rst/statistics_backend/get_info.rst

```diff
@@ -0,0 +1,94 @@
+.. include:: ../exports/alias.include
+
+.. _statistics_backend_get_info:
+
+Get entity meta information
+---------------------------
+
+.. warning::
+    This feature is currently not supported.
+    It will be implemented on a future release of *Fast DDS Statistics Backend*.
+
+*Fast DDS Statistics Backend* includes the possibility of retrieving the meta information of any given entities present
+in the network.
+Depending on the |EntityKind-api|, the returned tree contains merely the ``name``, or extra information such as ``pid``,
+``qos``, or ``data_type``.
+|get_info-api| returns a |Info-api| object.
+
+.. literalinclude:: /code/StatisticsBackendTests.cpp
+    :language: c++
+    :start-after: //CONF-GET-QOS-EXAMPLE
+    :end-before: //!
+    :dedent: 8
+
+.. _statistics_backend_get_info_host:
+
+Host Info example
+^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: /code/host_info_example.json
+    :language: JSON
+
+.. _statistics_backend_get_info_user:
+
+User Info example
+^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: /code/user_info_example.json
+    :language: JSON
+
+.. _statistics_backend_get_info_process:
+
+Process Info example
+^^^^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: /code/process_info_example.json
+    :language: JSON
+
+.. _statistics_backend_get_info_locator:
+
+Locator Info example
+^^^^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: /code/locator_info_example.json
+    :language: JSON
+
+.. _statistics_backend_get_info_domain:
+
+Domain Info example
+^^^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: /code/domain_info_example.json
+    :language: JSON
+
+.. _statistics_backend_get_info_participant:
+
+Participant Info example
```

Comment:
I would suggest to explain and show explicitly that the QoS is a `QoS`==`json` object, and not write down the QoS itself.
Maybe a new point in this section explaining the QoS structure.
(I cannot add this comment in the json file related: `/code/participant_info_example.json`.

Reply:
These are pretty set on stone. Removing any of them from Fast DDS would be a major version change in Fast DDS and here as well. I put them to have them as a guide for user to know what they can see beforehand.